torch_xla/csrc/ir.h

```diff
@@ -153,6 +155,9 @@ class Node {
 
   virtual XlaOpVector Lower(LoweringContext* loctx) const;
 
+ protected:
+  XlaOpVector ReturnOp(xla::XlaOp op, LoweringContext* loctx) const;
```

Comment:
I find this name misleading. The main thing it does it to associate `op` with the node, maybe call it `BindOp` or `AssociateOp`?

Reply:
That really is a return-from-lower-function helper, so that every operator that returns one op only, can avoid repeating the adding to the context, and the creation of the vector.

Reply:
I still don't like the name, it doesn't describe the main thing it does, which is the association between the node and the XLA op. The fact that it returns its input (in a container) is only there to make the interface fluent. It doesn't matter that much, though.

Reply:
It does that, and allows to return the vector required to be returned by the Lower() API.
BindOpAndReturn()?